src/kv/deserialise.h

```diff
@@ -135,6 +138,16 @@ namespace ccf::kv
           return ApplyResult::FAIL;
         }
 
+        if (
+          changes.size() == 3 &&
```

Comment:
duplicates check on line 132

Reply:
Actually it's not. Adding missing COSE sig check can't be done above because we have to support old ledgers. This one specifically checks that if size == 3, then the third one is COSE signature.